src/main/java/org/jcodec/containers/dpx/FilmInformationHeader.java

```diff
@@ -0,0 +1,11 @@
+package org.jcodec.containers.dpx;
+
+public class FilmInformationHeader {
```

Comment:
i would call this class `FilmHeader` because `Info` does not add meaning and takes up space